benchmarks/microbenchmarks/_plan.py

```diff
@@ -61,7 +61,7 @@ def run(self):
     def _get_kwargs(self, tags):
         kwargs = {}
         tags = tags[1:]  # tags = [case_name, item1_tag, item2_tag, ...]
-        for item, tag in zip(self.items, tags):
+        for item, tag in zip(self.items.values(), tags):
             kwargs[item.name] = item.impl(tag)
```

Comment:
If we write a new function like this function with this line changed to `kwargs[item.name] = tag` then we may not need `self.items[xxx].tag()`?

Reply:
Got it, and fixed in fecd219